README.rst

```diff
@@ -1,5 +1,16 @@
-Django Form Designer
-********************
+Django Form Designer (AI Fork)
+******************************
+
+Acknowledgements
+================
+
+This project is a fork of https://github.com/samluescher/django-form-designer .
```

Comment:
AFAIK the space between url and period is unneeded (Github renders the url without the period).